src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const ul = document.querySelector('ul');
+const listInfo = ul.querySelectorAll('li');
+// create a dataBase
+const dataBase = [];
+
+// add a data to dataBase
+for (const li of listInfo) {
+  dataBase.push(li);
```

Comment:
Currently, the `dataBase` array is populated with the entire `li` elements. If you only need specific data from each `li`, consider extracting that data before pushing it to the `dataBase`.